standalone-metastore/metastore-common/src/main/java/org/apache/hadoop/hive/metastore/HiveMetaStoreClient.java

```diff
@@ -790,6 +790,9 @@ public void close() {
     try {
       if (null != client) {
         client.shutdown();
+        if ((transport == null) || !transport.isOpen()) {
+          LOG.info("Closed a connection to metastore, current connections: " + connCount.decrementAndGet());
```

Comment:
You can use substitution while logging like it  is in other place in method. If possible please add a test for counter.

Reply:
An existing test, TestMetaStoreMetrics.testConnections is used for testing the counter.